andela/gryffindor-rc
# Feature/#151297879/integrate paystack

<img width="1210" alt="screen shot 2017-10-05 at 6 05 29 am" src="https://user-images.githubusercontent.com/16901595/31211716-55ec321a-a993-11e7-86d3-8e787fe972c8.png">

#### What does this PR do?
The PR includes integration for paystack. It allows users checkout with a paystack option.

#### Description of Task to be completed?
As a user, I should be able to make payments at checkout with paystack.

#### How should this be manually tested?
In development, follow the checkout flow on the user interface, use the dummy credentials provided by paystack and the transaction should be successful.

#### Any background context you want to provide?
This is a part of an EPIC which is to integrate other forms of payment.

#### What are the relevant pivotal tracker stories?
Integrate payment with Paystack- #151641614

#### Screenshots
<img width="1277" alt="screen shot 2017-10-05 at 6 03 05 am" src="https://user-images.githubusercontent.com/16901595/31211703-45f30528-a993-11e7-9fa4-fab82298a45a.png">
